corehq/apps/registration/static/registration/js/register_new_user.js

```diff
@@ -12,7 +12,7 @@ hqDefine('registration/js/register_new_user', [
     _,
     newUser,
     initialPageData,
-    kissmetrics
+    kissmetrics 
```

Comment:
there shouldn't be a diff on this line? can you remove the whitespace at the end please?

Reply:
Silly mistake 😅 
I will try to avoid it in the future.